<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShareFileDialog</class>
 <widget class="QDialog" name="ShareFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Shares</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="shareListView"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="newShareButton">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="removeShareButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Share Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Share Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="shareNameEdit">
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Place:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="fileNameEdit"/>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="chooseDirectoryButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QRadioButton" name="everyoneRadioButton">
        <property name="text">
         <string>Share to Everyone</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QRadioButton" name="specificUserRadioButton">
        <property name="text">
         <string>Specific Users only</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QListView" name="specificUserListView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShareFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShareFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
